### ISSUE TITLE
content_hash query param at like / pass

### ISSUE THREAD

**Issue by gregbarcza** [tinderjs/tinderjs#41]
Hello guys,

Recently the iOS tinder client started to add a query param named `content_hash` when you like or pass someone

for eg: 
`https://api.gotinder.com/like/548a2d2be871a40a305e7a80?content_hash=JDlSVoiXpsNaSoZTQQHGrIqmf5oh19U2Gfq5C5mImni57Hp`

 Maybe we should add this feature, since I guess they will be identify "third party clients" based on this soon.

What do you think?

**Comment by mayeaux** [comment]
yup agreed. I think the old API will be supported because of old clients, but let's see, if we can figure out how to use the new API I think that's the way forward for sure

**Comment by gregbarcza** [comment]
https://github.com/tinderjs/tinderjs/pull/42
